projects/client/RabbitMQ.Client/src/client/impl/ModelBase.cs

```diff
@@ -1,1567 +1,1571 @@
-                    Constants.ReplySuccess,
-                    "Nacks Received", new IOException("nack received")),
-                    false);
-                throw new IOException("Nacks Received");
-            }
-            if (timedOut)
-            {
-                Close(new ShutdownEventArgs(ShutdownInitiator.Application,
-                    Constants.ReplySuccess,
-                    "Timed out waiting for acks",
-                    new IOException("timed out waiting for acks")),
-                    false);
-                throw new IOException("Timed out waiting for acks");
-            }
-        }
-
-        protected virtual void handleAckNack(ulong deliveryTag, bool multiple, bool isNack)
-        {
-            lock (m_unconfirmedSet.SyncRoot)
-            {
-                if (multiple)
-                {
-                    for (ulong i = m_unconfirmedSet[0]; i <= deliveryTag; i++)
-                    {
-                        // removes potential duplicates
-                        while (m_unconfirmedSet.Remove(i))
-                        {
-                        }
-                    }
-                }
-                else
-                {
-                    while (m_unconfirmedSet.Remove(deliveryTag))
-                    {
-                    }
-                }
-                m_onlyAcksReceived = m_onlyAcksReceived && !isNack;
-                if (m_unconfirmedSet.Count == 0)
-                {
-                    Monitor.Pulse(m_unconfirmedSet.SyncRoot);
-                }
-            }
-        }
-
-        private QueueDeclareOk QueueDeclare(string queue, bool passive, bool durable, bool exclusive,
-            bool autoDelete, IDictionary<string, object> arguments)
-        {
-            var k = new QueueDeclareRpcContinuation();
-            Enqueue(k);
-            _Private_QueueDeclare(queue, passive, durable, exclusive, autoDelete, false, arguments);
-            k.GetReply(this.ContinuationTimeout);
-            return k.m_result;
-        }
-
-        public class BasicConsumerRpcContinuation : SimpleBlockingRpcContinuation
-        {
-            public IBasicConsumer m_consumer;
-            public string m_consumerTag;
-        }
-
-        public class BasicGetRpcContinuation : SimpleBlockingRpcContinuation
-        {
-            public BasicGetResult m_result;
-        }
-
-        public class ConnectionOpenContinuation : SimpleBlockingRpcContinuation
-        {
-            public string m_host;
-            public string m_knownHosts;
-            public bool m_redirect;
-        }
-
-        public class ConnectionStartRpcContinuation : SimpleBlockingRpcContinuation
-        {
-            public ConnectionSecureOrTune m_result;
-        }
-
-        public class QueueDeclareRpcContinuation : SimpleBlockingRpcContinuation
-        {
-            public QueueDeclareOk m_result;
-        }
-    }
-}
+// This source code is dual-licensed under the Apache License, version

```

Comment:
This file should not be showing this many differences. Is there an issue with newlines?